src/mgmt/pool.pb-c.c

```diff
@@ -1822,20 +1822,33 @@ static const ProtobufCFieldDescriptor mgmt__pool_destroy_req__field_descriptors[
     offsetof(Mgmt__PoolDestroyReq, svc_ranks),
     NULL,
     NULL,
-    0 | PROTOBUF_C_FIELD_FLAG_PACKED,             /* flags */
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)